src/repository/crop.repository.ts

```diff
@@ -36,3 +37,89 @@ export const deleteCrop = async (body: CropDeleteRequestBody) => {
     },
   });
 };
+
+export const getTargetCropTypeAndPlantedDate = async (
+  body: CropIdRequestBody
+) => {
+  return await prisma.crops.findUnique({
+    select: {
+      type: true,
+      planted_at: true,
+    },
+    where: {
+      id_user_id: {
+        id: body.cropId,
+        user_id: body.userId,
+      },
+    },
+  });
+};
+
+export const getConsecutiveAttendanceDays = async (
+  body: CropIdRequestBody,
```

Comment:
body의 userId만 사용되기 때문에 userId만 전달하도록 하는게 좋겠어요.
```suggestion
  userId: string,
```